src/test/components/FlameGraph.test.js

```diff
@@ -94,6 +94,9 @@ describe('FlameGraph', function() {
     const { getByText } = within(tooltip);
     const resourceLabel = getByText('Resource:');
     const valueElement = ensureExists(resourceLabel.nextSibling);
+
+    // See https://github.com/testing-library/jest-dom/issues/306
+    // eslint-disable-next-line jest-dom/prefer-to-have-text-content
```

Comment:
This one the only place where the autofixing was failing the test. Hopefully this will be resolved soon...